cpu/em_cpuhotplug.py

```diff
@@ -46,23 +48,23 @@ def setUp(self):
         self.T_CORES = int(self.CORES) * int(self.SOCKETS)
         self.log.info(" Cores = %s and threads = %s "
                       % (self.T_CORES, self.THREADS))
-        process.system("echo 8 > /proc/sys/kernel/printk", shell=True,
-                       ignore_status=True)
+
+        genio.write_one_line('/proc/sys/kernel/printk', "8")
         self.max_smt = 4
         if cpu.get_cpu_arch().lower() == 'power8':
             self.max_smt = 8
         if cpu.get_cpu_arch().lower() == 'power6':
             self.max_smt = 2
-        process.system_output("ppc64_cpu --smt=%s" % self.max_smt, shell=True)
+        process.system("ppc64_cpu --smt=%s" % self.max_smt, shell=True)
```

Comment:
what is the reason for change ?

Reply:
as system_output is not needed here as we not capturing it any where